### ISSUE TITLE
include `date_is_reliable` is topic CSV download?

### ISSUE THREAD

**Issue by rahulbot** [mediacloud/web-tools#1647]
Aashka is wondering if we can help give better clues for when dates can be trusted and when not. For instance, in her UN SDGs topic she saw a spike on a date, but it turned out to be just many stories incorrectly dated.

We use the `date_is_reliable` column to show story dates in the web interface in italics with a "?" after them. Should we include this Boolean variable in the download CSV? @hroberts how is this attribute filed in? What does it mean?

**Comment by hroberts** [comment]
we add that flag unless we got the date from an rss feed (or some similar
structured syndication format) or is one of a small number of date guessing
methods that we consider to be very reliable (such as a date stub in the
url).  for all other methods combined, our date guessing is around 80%
accurate to within a day.  the best defense for this is just for aashka to
have a sense that if you see a big weird date spike, she should consider
date guessing as the cause.

in theory, we could add some sort of monitor to look for date spikes within
topics (they all come from topics because we only use date guessing for
spidered stories).  but I'm not sure what we would do once we found that
spike other than warn the user.  we're already doing the best we can at the
date guessing, so there's nothing more than we can do without manual
intervention.

-hal
